pkgs/os-specific/darwin/apple-source-releases/system_cmds/fix-implicit-declarations.patch

```diff
@@ -0,0 +1,48 @@
+diff -ur a/getty.tproj/main.c b/getty.tproj/main.c
+--- a/getty.tproj/main.c	2008-06-10 14:50:19.000000000 -0400
++++ b/getty.tproj/main.c	2023-05-31 18:06:40.121028558 -0400
+@@ -67,6 +67,7 @@
+ #include <syslog.h>
+ #include <termios.h>
+ #include <time.h>
++#include <util.h>
+ #include <unistd.h>
+ 
+ #ifdef __APPLE__
+@@ -152,7 +153,7 @@
+ static void	putpad(const char *);
+ static void	puts(const char *);
+ static void	timeoverrun(int);
+-static char	*getline(int);
++static char	*get_line(int);
```

Comment:
I can't figure out why this rename? Apple's `util.h` doesn't seem to define either version.

Reply:
`<util.h>` includes `<stdio.h>`, which defines `getline`.

Reply:
I updated the commit message to explain the renaming.